pages/implementation/avo-tracking-plan-webhook.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: 'guide'
+title: 'Subscribe to tracking plan changes webhook'
+---
+
+import Link from '../../components/Link';
+
+After setting up Webhook integration in <Link passHref href="/workspace/tracking-plan/publishing"><a>Tracking Plan Publishing</a></Link>, your endpoint provided there will be called with a [JSON
+Schema](https://json-schema.org) payload representing the recent snapshot of your tracking plan when your tracking plan is published.
+
+You can examine and download the exact JSON in the Publishing tab of your [Avo workspace](https://www.avo.app/schemas/).
+
+The format is similar to:
+
+```JSON
+{
+  "name": "Avo",
+  "events": [
+    {
+      "name": "Debugger Started",
+      "description": "Sent when the web debugger is started.",
+      "rules": {
+        "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
The format is documented in the publishing docs, should we link to there instead?